README.md

```diff
@@ -75,3 +75,29 @@ Most important performance modifications include:
 * avoiding expensive calls of `model.frame()` and `model.matrix` and using the design matrix and response vector instead of a data.frame and a formula
 * using `lm.fit()` instead of `lm()` when possible
 
+## Contribution guidelines:
+We always welcome contributions to the package.
+
+### When you are familiar with GitHub:
+We work with [GitHub Flow](https://guides.github.com/introduction/flow/) branching model.
+
+Key steps:
+1. Create a GitHub issue in this repository with description of the work that you plan to do.
+2. Assign yourself to the GitHub issue you are working on, to inform other developers that you are working on it.
+3. Create your own working branch (or fork) based on the `dev` branch.
+4. Make your changes in that branch or fork.
+5. Commit your changes to your working branch/fork as long as you are not finished with your development.
+6. Make sure the release notes in NEWS.Rd are updated.
```

Comment:
Not sure we need that; eventually we should, but we also have the git commit history and closed issues, so I feel that this may be effort duplication.

Reply:
Do you refer to the last point or all points?

Reply:
Always the last point, GitHub formats these in a way that it also shows 3 lines above.

Reply:
Having a [changelog](https://guide.esciencecenter.nl/best_practices/releases.html) is a good practice. Users would like to see a concise summary of notable changes in a new release without going through detailed history, issues etc.